package.json

```diff
@@ -36,7 +36,7 @@
     "moment": "^2.17.1"
   },
   "devDependencies": {
-    "@angular/cli": "^1.0.0",
+    "@angular/cli": "1.0.0",
```

Comment:
[Yes](https://github.com/angular/angular-cli/issues/2034#issuecomment-297274534).  I can put in a workaround for the angular-cli bug.  There are two workaround options:

1. Split all interfaces into separate files
-OR-
2. Do this:
```ts
  // tslint:disable-next-line:typedef
  @Input() type = <CalendarType>'day';
```

instead of:
```ts
  @Input() type: CalendarType = 'day';
```


Should I fix this in a separate PR?